tools/ci/tc/tasks/test.yml

```diff
@@ -188,10 +188,16 @@ tasks:
                 - trigger-push
             - vars:
                 browser: chrome
-                channel: dev
+                channel: canary
               use:
                 - trigger-master
                 - trigger-push
+            - vars:
+                browser: chrome
+                channel: dev
+              use:
+                - trigger-weekly
+                - trigger-push
```

Comment:
@jgraham are you OK with us runner both Canary and Dev on every push for about a week so that we can score both the end of Interop 2023 with Dev and the start of Interop 2024 with Canary?

Reply:
Note that the experimental label is added in the results processing phase in the wpt.fyi code. We will be able to choose when to switch to Canary as experimental when we land [this change](https://github.com/web-platform-tests/wpt.fyi/pull/3673)